vg/draw/canvas.go

```diff
@@ -276,24 +314,59 @@ func (c *Canvas) Y(y float64) vg.Length {
 	return vg.Length(y)*(c.Max.Y-c.Min.Y) + c.Min.Y
 }
 
-// Crop returns a new Canvas corresponding to the receiver
-// area with the given number of inches added to the minimum
+// Crop returns a new Canvas corresponding to the Canvas
+// c with the given lengths added to the minimum
 // and maximum x and y values of the Canvas's Rectangle.
-func (c Canvas) Crop(minx, miny, maxx, maxy vg.Length) Canvas {
+// Note that cropping the right and top sides of the canvas
+// requires specifying negative values of right and top.
+func Crop(c Canvas, right, bottom, left, top vg.Length) Canvas {
```

Comment:
I like the new `right`, `bottom`, `left` and `top` names.
that's really, *really* nitpicking, but: is it the right order though?
I don't have a preference and, in that department, consistency is king.

I suppose you chose a clockwise order.
wouldn't `left, right, bottom, top` make more sense to left-right reading people? (sorry `<right-to-left languages>`)

(feel free to ignore me, though)